firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -1039,16 +1039,15 @@ class BrowserCoordinator: BaseCoordinator,
     }
 
     func showNativeErrorPage(overlayManager: OverlayModeManager) {
-        // TODO: FXIOS-9641 #21239 Integration with Redux - presenting view
-        let errorpageController = self.errorViewController ?? NativeErrorPageViewController(
+        let errorPageController = self.errorViewController ?? NativeErrorPageViewController(
```

Comment:
@lmarceau I remember we discussed on trying to remove these references on `BrowserCoordinator`. I tried to address this for private homepage since it's a simple view that doesn't contain variable data. See PR: https://github.com/mozilla-mobile/firefox-ios/pull/24358

Could we do something similar here? Or is there a specific reason we need to keep the reference?